modules/ting_infomedia/ting_infomedia.install

```diff
@@ -65,13 +46,8 @@ function ting_infomedia_update_7000() {
 }
 
 /**
- * Unlock ding_entity_field to allow deletion of field
+ * Unlock fields, see ding_entity_unlock_fields().
  */
 function ting_infomedia_update_7001() {
-  db_update('field_config')
```

Comment:
The new version does the same as the old, but in the same way as all the other new update functions. The functionality haven't changed, the implementation just adjusted for consistency.